contracts/Tokenomics.sol

```diff
@@ -1296,6 +1296,46 @@ contract Tokenomics is TokenomicsConstants {
         return true;
     }
 
+    /// @dev Updates inflation per second if .
+    function updateInflationPerSecond() external {
+        // Check for the contract ownership
+        if (msg.sender != owner) {
+            revert OwnerOnly(msg.sender, owner);
+        }
+
+        // Get current epoch length
+        uint256 curEpochLen = epochLen;
+        // Find if the year changes within the epoch
+        uint256 numYears = (block.timestamp + curEpochLen - timeLaunch) / ONE_YEAR;
+        // Revert if the year changes within the next epoch as it requires more complicated set of calculations
+        if (numYears > currentYear) {
+            revert Overflow(numYears, currentYear);
+        }
+
+        // Ensure effectiveBond is bigger than maxBond
+        uint256 curEffectiveBond = effectiveBond;
+        uint256 curMaxBond = maxBond;
+        if (curMaxBond > curEffectiveBond) {
+            revert Overflow(curMaxBond, curEffectiveBond);
+        }
+
+        // Adjust effective bond by reducing it with the on-going epoch maxBond value
+        curEffectiveBond -= curMaxBond;
+
+        // Recalculate inflation per second based on the updated current year inflation
```

Comment:
If the curve hasn't changed, the value will be the same as before right? We should have a test

Reply:
Correct

Reply:
Let's add this as a comment to the contract and also write a test for it